compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
```

Comment:
do you want to put this in a peer dir, like compiler_opt/<name>/generate... etc?

Reply:
Yes, we can create a new dir but let's do that once we are done with the trajectory generating code. To that dir we can add the code for training the model and re-weighting the trajectories.

Reply:
sounds good - can you open an issue? no need to tag it in files because it'd be super noisy

Reply:
Done.